app/src/main/java/org/schabi/newpipe/util/SecondaryStreamHelper.java

```diff
@@ -29,33 +31,37 @@ public SecondaryStreamHelper(final StreamSizeWrapper<T> streams, final T selecte
      * @param videoStream  desired video ONLY stream
      * @return selected audio stream or null if a candidate was not found
      */
+    @Nullable
     public static AudioStream getAudioStreamFor(@NonNull final List<AudioStream> audioStreams,
                                                 @NonNull final VideoStream videoStream) {
-        switch (videoStream.getFormat()) {
-            case WEBM:
-            case MPEG_4:// ¿is mpeg-4 DASH?
-                break;
-            default:
-                return null;
-        }
+        final MediaFormat mediaFormat = videoStream.getFormat();
+        if (mediaFormat != null) {
+            switch (mediaFormat) {
+                case WEBM:
+                case MPEG_4:// ¿is mpeg-4 DASH?
```

Comment:
I guess this should be resolved, shouldn't it?

Reply:
This isn't my code, I just reformatted it.